plugins/linux-capture/CMakeLists.txt

```diff
@@ -46,8 +46,14 @@ set(linux-capture_LIBRARIES
 
 option(ENABLE_PIPEWIRE "Enable PipeWire support" ON)
 if(ENABLE_PIPEWIRE)
-	find_package(PipeWire REQUIRED)
-	find_package(Gio REQUIRED)
+	find_package(PipeWire QUIET)
+	find_package(Gio QUIET)
+
+	if(NOT PIPEWIRE_FOUND)
+		message(FATAL_ERROR "PipeWire library not found, Please install PipeWire or set ENABLE_PIPEWIRE=OFF")
```

Comment:
There's a capitalization issue here and in the message below. Either change the comma to a dot, or de-capitalize `Please` :slightly_smiling_face: